website/src/app.jsx

```diff
@@ -11,6 +11,12 @@ export default React.createClass({ // eslint-disable-line
     return Object.assign({}, initialState)
   },
 
+  componentWillUpdate() {
+    const {refs: {maskedInput}} = this
+
+    maskedInput.setState(maskedInput.getInitialState())
```

Comment:
@browniefed Even if we make things controlled in the website, the component will have `conformedInput` as a state, we need to clear that for the new choices to work correctly.